nodejs/examples/webhook-server/README.md

```diff
@@ -0,0 +1,27 @@
+# NodeJS webhook server example
+A http server than can receive and verify signed TrueLayer webhooks.
+
+## Run
+
+First, build the signing lib
+```sh
+# in the `truelayer-signing/nodejs` dir
+yarn build
```

Comment:
Isn't it just `yarn`?

```suggestion
yarn
```

Reply:
Maybe that works, but `build` is the specific script that does actually builds the lib.js file:
https://github.com/TrueLayer/truelayer-signing/blob/main/nodejs/package.json#L9

Reply:
Ah I see the directory, wouldn't just running `yarn` in the examples/webhook-server dir work fine in any case?

yarn is playing up for me at the moment, but `npm i` in the webhook-server does the right thing.

Reply:
Judging by @jdon's comment in the slack thread, it doesn't seem to work properly (it will pull the path dependency, but doesn't run the build script to build)